_pages/guides/react.markdown

```diff
@@ -203,3 +203,39 @@ You can extract additional information from MarkLogic's Search API by adding an
     ```
 
     Though, note that you could also combine several extracts together.
+
+### Add an image to the top Navbar
+
+You may prefer to have an image in the top left corner of the navigation bar.  Grove's MLNavbar component was designed with this in mind and can be customized with a prop (or two if you'd like to style the image, too).
+
+1. **Open `ui/src/components/Navbar.js`**  A custom Navbar component that renders an MLNavbar child component was provided as part of the Grove installation.
+
+2. **Copy the image to `ui/src/components`** For this example, `Logo.png` was copied into this directory.
+
+3. **Import the image**  Because this project uses Webpack, you must import a file directly into the module so that it will be included properly in the Webpack bundle.  In this example, importing a file provides a string value for the final path so that the `src` attribute of an image can properly locate the file.
+
+    ```javascript
+      import myLogo from './Logo.png';
+    ```
+
+4. **Pass the image as a prop to MLNavbar**  Add the `logo` prop to MLNavbar, passing `myLogo` to this child component.
+
+    ```javascript
+      <MLNavbar {...props} brandLink={brandLink} logo={myLogo}>
+    ```
+    Go back to browser to inspect the result.  The default position may be suitable for your needs.  If it is not, continue to the next step.
+
+5.  **(Optional) Style the image with an inline style**  The MLNavbar component also has a `logoStyle` prop.  This prop allows you to pass down an inline style object to assist with tasks such as sizing or positioning the image.  You may need to customize the styling to suit your needs and properly fit the image.  Note that a React inline style object uses camelCased properties rather than a CSS string.  The following is a simple example that restricts the size of the image and moves it slightly to align it better with the navigation options.
+
+    ```javascript
+      <MLNavbar
+        {...props}
+        brandLink={brandLink}
+        logo={mlLogo}
+        logoStyle={ {marginTop: '-5px', maxWidth: '100px', maxHeight: '45px'} }
```

Comment:
Would this be a better default than the default styling now? I think I may have picked the original styling to fit a MarkLogic logo that I had. But your styling worked well for the Grove logo file I picked to test.

Reply:
It might be.  I picked this style to work with the white MarkLogic logo.  It aligned well with the Search and Create text in the Navbar.  I'll add a ticket to make this the default style for MLNavbar.